src-tauri/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "wooting-macros"
-version = "1.0.2"
+version = "1.1.0-pre-release"
 description = "Macro software made by Wooting."
-authors = ["Jeroen N", "Erik S", "Simon W", "David M", "Khang N"]
+authors = ["Jeroen N", "Erik S", "Simon W", "David M <1900179+medzernik@users.noreply.github.com>", "Khang N", "Pola M"]
```

Comment:
Added my email (since it seems I'll maintain the app long-term), but maybe i should add others as well?

Reply:
Do you know if these details end up in the compiled binary? If they don't, then I don't think they are really needed

Reply:
https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates

This is the most I found, I think it could. In any case, the more points of contact to us, the easier to contact us when someone forks the app for example was my thinking. I can also remove it tho if you think it's not neccessary

Reply:
There's not really any harm in it, but it is kinda pointless. In the case where someone forks the app, they'll be able to find your github profile pretty easily and contact you that way if needed.

Reply:
Removed: [094952f](https://github.com/WootingKb/wooting-macros/pull/194/commits/094952f6f2095197929d5ed9c37b4cb42cff37bc)